### PR TITLE
Feature/Require TLS (Laravel 4.2 Branch)

### PR DESCRIPTION
**Pull request by AdrianHL**
I found that the Laravel 4.2 version does not include a function to trigger the Require TLS parameter per-message basis. It'd great to have in the next 3.1 version.

### CODE REVIEW

File: src/Bogardo/Mailgun/Mailgun.php [hunk 1]
```diff
@@ -98,7 +98,7 @@ public function send($view, array $data, $callback, $mustInit = true)
 
 		$this->getMessage($view, $data);
 
-		return $this->mailgun()->sendMessage(Config::get('mailgun::domain'), $this->getMessageData(), $this->getAttachmentData());
+		return $this->mailgun(true, Config::get('mailgun::api_key'))->sendMessage(Config::get('mailgun::domain'), $this->getMessageData(), $this->getAttachmentData());
```
Comment (Bogardo) [inline src/Bogardo/Mailgun/Mailgun.php]: Should this be part of the PR?

Comment (AdrianHL) [inline src/Bogardo/Mailgun/Mailgun.php]: No. It was automatically picked when I selected the Bogardo:develop branch. Thanks

**Comment by AdrianHL** [comment]
No. It was automatically picked when I selected the Bogardo:develop branch. Thanks

**Comment by AdrianHL** [comment]
Hi @Bogardo,

Any chance to get this pull request merged.

Thanks in advance.

**Comment by abbatis** [comment]
We're dropping support for all older Laravel versions.

**Pull request closed without merging**